app/src/main/java/com/github/pockethub/android/core/commit/CommitUtils.kt

```diff
@@ -24,7 +24,7 @@ import com.meisolsson.githubsdk.model.Commit
 import com.meisolsson.githubsdk.model.GitHubFile
 import com.meisolsson.githubsdk.model.git.GitCommit
 import java.text.NumberFormat
-import java.util.Date
```

Comment:
Why did we change this?

Reply:
Android Studio automatically replaces `java.util` imports with `java.util.*`. Guess I can revert it.